tldr.py

```diff
@@ -28,7 +28,7 @@
 ).rstrip('/')
 DOWNLOAD_CACHE_LOCATION = os.environ.get(
     'TLDR_DOWNLOAD_CACHE_LOCATION',
-    'https://tldr-pages.github.io/assets/tldr.zip'
+    'https://tldr.sh/assets/tldr.zip'
```

Comment:
We probably need to switch this to use releases to comply with the latest specification.

Reply:
```suggestion
    'https://github.com/tldr-pages/tldr/releases/latest/download/tldr.zip'
```